lib/si-events-rs/src/audit_log/v1.rs

```diff
@@ -1,35 +1,134 @@
 use serde::{Deserialize, Serialize};
-use strum::{AsRefStr, Display, EnumIter, EnumString};
+use strum::{Display, EnumDiscriminants};
 
-use crate::Actor;
+use crate::{
+    ActionKind, ActionPrototypeId, Actor, AttributeValueId, ChangeSetId, ComponentId, FuncId,
+    InputSocketId, OutputSocketId, PropId, SchemaVariantId, SecretId,
+};
 
-#[derive(Clone, Debug, Deserialize, Serialize, Eq, PartialEq)]
+#[derive(Debug, Deserialize, Serialize, Clone, PartialEq, Eq)]
 pub struct AuditLogV1 {
     pub actor: Actor,
     pub kind: AuditLogKindV1,
+    pub entity_name: String,
     pub timestamp: String,
+    pub change_set_id: Option<ChangeSetId>,
 }
 
 #[remain::sorted]
-#[derive(
-    Clone,
-    Debug,
-    Deserialize,
-    Serialize,
-    Eq,
-    PartialEq,
-    Hash,
-    AsRefStr,
-    Display,
-    EnumIter,
-    EnumString,
-)]
+#[derive(Debug, Deserialize, Serialize, Clone, PartialEq, Eq, Display, EnumDiscriminants)]
 pub enum AuditLogKindV1 {
```

Comment:
This looks good--and even convenient to see all the possibilities together. I wonder long term if all the embedded fields will push the size of the enum to be larger than expected. Although for this to happen, we'd have to have a variant with many more fields or a field with a large stack size type. Anyway, nothing actionable, just reflecting out loud 😄

Reply:
Thank you! I think this is going to be a pain soon enough. I'm okay with it for now due to the amount of escape hatches we have available to us, but yes, you are right. It's going to get huge.